tests/acs-security-demos-no-routes/diff_output_from_acs-security-demos.txt

```diff
@@ -1,3 +1,3 @@
 Connectivity diff:
-source: {ingress-controller}, destination: frontend/asset-cache[Deployment], dir1:  TCP 8080, dir2: No Connections, diff-type: removed
-source: {ingress-controller}, destination: frontend/webapp[Deployment], dir1:  TCP 8080, dir2: No Connections, diff-type: removed
\ No newline at end of file
+source: {ingress-controller}, destination: frontend/asset-cache[Deployment], dir1:  TCP 8080, dir2: No Connections, diff-type: removed (workload {ingress-controller} removed)
+source: {ingress-controller}, destination: frontend/webapp[Deployment], dir1:  TCP 8080, dir2: No Connections, diff-type: removed (workload {ingress-controller} removed)
```

Comment:
`ingress-controller` is not an added/removed workload.. should be an exception (its a symbolic notation for actual ingress controller pod, and its deployment does not depend if there is or not a Route or Ingress resources).

Reply:
fixed